www/faq.php

```diff
@@ -941,15 +943,15 @@
    </td></tr>
 
 <tr id="abi">
-<?php echo freshports_PageBannerText('Why isthis particular ABI not listed on the website??'); ?>
+<?php echo freshports_PageBannerText('Why is this particular ABI not listed on the website?'); ?>
 </tr>
 
    <tr><td class="textcontent">
    <p>FreshPorts covers the ABIs published by the project at <a href="https://pkg.freebsd.org/index.html">https://pkg.freebsd.org/index.html</a>.</p>
 
    <p>In general, the project does not build quarterly branches for whatever branch is now CURRENT.</p>
```

Comment:
This paragraph might be superfluous. 

<https://dev.freshports.org/faq.php#packages> (above) states "… No packages are built for that ABI/package set combination.".  

Whilst the foot of <https://dev.freshports.org/--/package-imports.php?sort=name> does, currently, show six _quarterly_ rows for FreeBSD:15:⋯, those peculiarities are not evident in this section of the FAQ page.

Reply:
I'll leave the superfluous.  People often read only one FAQ.

... I think your last sentence asks: If we don't build quarterly for current, why is FreshPorts showing that?

The answer is: it's easier than figuring out what is `current`.